src/util/Enums.js

```diff
@@ -79,4 +79,10 @@ export default {
   UNIVERSAL_LINK_CHALLENGE: 'UNIVERSAL_LINK',
 
   LAUNCH_AUTHENTICATOR_REMEDIATION_NAME: 'launch-authenticator',
+
+  FORM_NAME_TO_OPERATION_MAP: {
```

Comment:
this var probably shall go into `RemediationConstant` but uses `Enums.PRIMARY_AUTH`, etc.